src/core/less/variables/colors.less

```diff
@@ -0,0 +1,220 @@
+// Main Colors
```

Comment:
i don't know whether the "module" specific stuff from files like this should be broken up into "modules/notify/colors.less" etc

Reply:
Done.